### PR TITLE
flaky-e2e-tests fix

### PR DESCRIPTION
**Pull request by dtfranz**
Fixing flake tests so they don't fail 100% of the time.

E2E tests were updated recently to use a generated namespace with an OperatorGroup created from the beginning of each test. However some of the tests were not updated to reflect this and were still creating and deleting resources in an additional namespace, or began to fail because the test environment required that no OperatorGroups exist from the beginning of the test.

This commit addresses two tests that have been failing at a rate of 100% due to the issues stated above.

### CODE REVIEW

File: test/e2e/installplan_e2e_test.go [hunk 1]
```diff
@@ -3223,17 +3208,11 @@ var _ = Describe("Install Plan", func() {
...
 
 		BeforeEach(func() {
-			ns = &corev1.Namespace{}
-			ns.SetName(genName("ns-"))
-
-			// Create a namespace
-			Eventually(func() error {
-				return ctx.Ctx().Client().Create(context.Background(), ns)
-			}, timeout, interval).Should(Succeed(), "could not create Namespace")
+			// Make sure there are no OGs in the namespace already
```
Comment (dtfranz) [inline test/e2e/installplan_e2e_test.go:3214]: OGs are created in the `generatedNamespace` before each test in this file but this particular test relies on a blank slate to work consistently. This is a quick-and-dirty way to make sure we get that clean slate.

File: test/e2e/installplan_e2e_test.go [hunk 2]
```diff
@@ -3247,13 +3226,12 @@ var _ = Describe("Install Plan", func() {
 			outIP.Status = ip.Status
 			_, err = crc.OperatorsV1alpha1().InstallPlans(generatedNamespace.GetName()).UpdateStatus(context.Background(), outIP, metav1.UpdateOptions{})
 			Expect(err).NotTo(HaveOccurred())
+
```
Comment (dinhxuanvu) [inline test/e2e/installplan_e2e_test.go:3229]: This is no needed.

File: test/e2e/installplan_e2e_test.go (hunk 1 above)
Comment (dinhxuanvu) [reply test/e2e/installplan_e2e_test.go:3214]: I understand the intention is to reuse one namespace all for tests under this umbrella but should you take care of the fact a new InstallPlan still being created for each test in the same namespace?

**Review by dinhxuanvu — CHANGES_REQUESTED** [review]

File: test/e2e/installplan_e2e_test.go (hunk 1 above)
Comment (dinhxuanvu) [reply test/e2e/installplan_e2e_test.go:3214]: Ignore this comment, the AfterEach should take care of this. So we are cool here.

File: test/e2e/installplan_e2e_test.go (hunk 2 above)
Comment (dtfranz) [reply test/e2e/installplan_e2e_test.go:3229]: Woops, yeah I forgot to push up that change.

**Review by dinhxuanvu — APPROVED** [review]
/approve
/lgtm

**Review by tmshort — APPROVED** [review]

**Pull request merged**